.yamllint

```diff
@@ -1,7 +1,6 @@
+---
+
 extends: relaxed
 
 rules:
-  # 80 chars should be enough, but don't fail if a line is longer
-  line-length:
-    max: 150
-    level: warning
+  line-length: disable
```

Comment:
**question**: why remove this?

Reply:
The nature of this project is there are a lot of longer variable names and the line-length has proven awkward to work with.